lib/internal/http2/compat.js

```diff
@@ -386,7 +386,10 @@ class Http2ServerResponse extends Stream {
     this.writeHead(statusCode);
   }
 
-  writeHead(statusCode, headers) {
+  writeHead(statusCode, statusMessage, headers) {
+    if (headers === undefined && typeof statusMessage === 'object') {
+      headers = statusMessage;
+    }
```

Comment:
Okay, updated. ✍️ (Feel free to rewrite the message. Not sure how verbose these comments ought to be.)